crates/accelerate/src/sabre_swap/neighbor_table.rs

```diff
@@ -97,10 +103,13 @@ impl NeighborTable {
     }
 
     fn __getstate__(&self) -> Vec<Vec<PhysicalQubit>> {
```

Comment:
For this the return type doesn't matter too much, it's more about just making sure we returns something pyo3 converts to native py types on return so that we can rebuild the state. I'm assuming that there are no smallvec conversions built into pyo3 yet, so we need to do this anyway. If this was at all performance sensitive (which it is not) then I'd suggest just returning a pylist directly to avoid a double iteration, but this is fine.

Reply:
Oddly, I thought I had done this in terms of `PyList`.  I wonder if I accidentally used a slightly older version of this commit as the PR.  I can change it over without any pain.

Reply:
Done in 5f313e22d.